app/views/layouts/application.html.erb

```diff
@@ -43,7 +43,8 @@
   <body <% if content_for?(:body_attributes) %>
           <%= yield(:body_attributes) %>
         <% end %> class="<%= yield(:body_classes) %>">
-    <% is_practice_editor = params[:controller] === 'practices' && (NavigationHelper::PRACTICE_EDITOR_PAGES.include?(params[:action]) || params[:action] === 'metrics') %>
+    <% is_practice_editor = params[:controller] === 'practices' && (NavigationHelper::INNOVATION_EDITOR_PAGES.include?(params[:action]) || params[:action] === 'metrics') %>
+    <% is_product_editor = params[:controller] === 'products' && NavigationHelper::INNOVATION_EDITOR_PAGES.include?(params[:action]) %>
```

Comment:
what do you think about moving these conditionals into the `NavigationHelper`? I think this layout has access to the `NavigationHelper` via the application controller.